### DISCUSSION TITLE
Rename "lightweight" and "litemode" file read options to "settings-only"?

### DISCUSSION THREAD

**Discussion by raph-luc** [ansys/pyfluent#2288] [Ideas]
From the very general "lightweight" to the more specific "settings-only"

Would require Fluent server-side changes

See https://github.com/ansys/pyfluent/pull/2280#discussion_r1409416051

**Comment by ansjsia** [comment]
I think we also need to make it very clear, especially in terminology, that there are three different ways to read a file:

- **Normal:** Load the mesh and settings.
- **Settings-only:** Do not load the mesh, but load settings.
- **Lightweight (PyFluent only):** Load settings only at first, then load the mesh in the background.

Presently, it seems like we have "settings-only" implemented but it is called "lightweight".
